src/main/java/org/zalando/nakadi/domain/Timeline.java

```diff
@@ -168,4 +172,22 @@ public boolean equals(final Object o) {
     public int hashCode() {
         return id != null ? id.hashCode() : 0;
     }
+
+    public static Timeline createTimeline(
+            final String eventType,
+            final int order,
+            final Storage storage,
+            final String topic,
+            final Date createdAt) {
+        final Timeline timeline = new Timeline(eventType, order, storage, topic, createdAt);
+        timeline.setId(UUID.randomUUID());
+        return timeline;
+    }
+
+    public static Timeline createFakeTimeline(final EventType eventType, final Storage storage) {
```

Comment:
@rcillo As the one who invented this, I will answer. There are several types of timelines (we are considering, that now there are no timelines at all)
1. No timeline for event type. We do not have database entity "Timeline", but in order to unify code - we will generate this fake object. The cursor format for fake timeline will be the same as current one (no version is added).
2. Very first timeline. No new topic is created for this timeline, (so no storage change actually is there), but database object is created, hence offset now includes version. And another thing - this timeline may be deleted, because underlying storage is still the same as in 1. 
3. Usual timeline. The only difference from 2 is that it can not be deleted, because different topic was used